### DISCUSSION TITLE
microtexture evolution of rolled alloys

### DISCUSSION THREAD

**Discussion by makeabhishek** [mtex-toolbox/mtex#1438] [Ask Anything]
Hi, I am trying to generate a microstructure with elongated rolled microstructure evolution. I saw that texture evolution can be controlled using mtex in this [example](https://mtex-toolbox.github.io/TaylorHex.html). Is it possible to generate grain statistics also and create 2D synthetic microstructure features? Attached is the image that I want to generate using mtex.

![example](https://user-images.githubusercontent.com/47937684/170471747-f5f48cb0-af78-4b44-944a-ded6a927d79e.JPG)

**Comment by kilir** [comment]
Hi,
no this is not implemented.
Cheers,
Rüdiger
